snooty/test_project.py

```diff
@@ -266,25 +230,23 @@ def test_recursive_source_constants() -> None:
 
 def test_target_wipe() -> None:
     backend = Backend()
-    with Project(
-        Path("test_data/test_postprocessor"), backend, {}, watch=False
-    ) as project:
-        project.build()
-        with project._lock:
-            query_result = project._project.targets["std:label:global-writes-zones"][0]
-            assert isinstance(query_result, TargetDatabase.InternalResult)
-            assert query_result.result[0] == "index"
-
-        project.update(
-            FileId("index.txt"),
-            ".. _gooblygooblygoo:\n",
-        )
-        project.postprocess()
-        with project._lock:
-            assert not project._project.targets["std:label:global-writes-zones"]
-            query_result = project._project.targets["std:label:gooblygooblygoo"][0]
-            assert isinstance(query_result, TargetDatabase.InternalResult)
-            assert query_result.result[0] == "index"
+    project = Project(Path("test_data/test_postprocessor"), backend, {}, watch=False)
```

Comment:
Looks like `watch` is no longer used for anything in Project. Should we keep the field defined (both here and within the class init)?

Reply:
Indeed! It is now unused